### PR TITLE
USER STORY - 06( continuining) - SIGN UP AS A GRADUATE( explaining in details for the implementation of  SENDING VERIFICATION EMAIL to the GRADS)

### PR DESCRIPTION
**Pull request by Adibab**
## Naming Rules

Name your PR like this: ISSUENUMBER-#14-IMPLEMENTATION OF SENDING VERIFICATION EMAIL  to the GRADS-ADIBA BELAYETE

## Description

I have added a form to provide the email id & certificate number by the grads so their certificate id can be verified & a verification email will be sent to the provided email address to confirm their identity. 

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:
 ISSUE NUMBER # 14

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented on my code
- [ ] I have updated any documentation

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -2,5 +2,7 @@
 	"plugins": [
```
Comment (Rafsanjani) [inline .babelrc:2]: You shouldn't touch this file

File: .vscode/settings.json [hunk 2]
```diff
@@ -2,6 +2,6 @@
 	"editor.formatOnSave": false,
 	"editor.codeActionsOnSave": {
 		"source.fixAll.eslint": true
-	},
-	"files.eol": "\n"
```
Comment (Rafsanjani) [inline .vscode/settings.json:6]: don't edit this file

File: client/components/users/VerifyCertificateNum.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
+.signUpInfoContainer {
```
Comment (Rafsanjani) [inline client/components/users/VerifyCertificateNum.css:1]: Please format this file

File: client/components/users/VerifyCertificateNum.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
```
Comment (Rafsanjani) [inline client/components/users/VerifyCertificateNum.js:1]: format with prettier

File: server/utils/config.js [hunk 5]
```diff
@@ -12,10 +12,10 @@ function createDatabaseUrl() {
 		return process.env.DATABASE_URL;
 	}
 	const host = process.env.DB_HOST ?? "localhost";
-	const name = process.env.DB_NAME ?? "cyf";
-	const password = process.env.DB_PASS ?? process.env.DB_PASSWORD ?? "";
+	const name = process.env.DB_NAME ?? "cyf_hotel";
```
Comment (Rafsanjani) [inline server/utils/config.js:15]: Undo all the changes in this file. Let's keep the defaults.

**Review by Rafsanjani — CHANGES_REQUESTED** [review]

**Review by Rafsanjani — CHANGES_REQUESTED** [review]
Format with prettier and it should be good to go

**Comment by Adibab** [comment]
Hi, I have done the formate files with prettier. thanks.

**Review by Rafsanjani — APPROVED** [review]

File: client/components/users/VerifyCertificateNum.js [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+		}
+	};
+
+	return (
+		<div className="signUpInfoContainer">
+			<ToastContainer position="bottom-center" limit={1} />
+			<h2 className="signUpHeader"> Sign Up As A Graduate </h2>
+			<form onSubmit={handleSubmit}>
+				<div>
+					<label for="email"> Email</label>
```
Comment (Rafsanjani) [inline client/components/users/VerifyCertificateNum.js:41]: This should be `<label htmlFor="email"> Email</label>`

File: client/src/App.js
Comment (Rafsanjani) [inline client/src/App.js:1]: Are you sure you want to modify this file? It looks like you are replacing all the items in here with your new component.

**Review by Rafsanjani — CHANGES_REQUESTED** [review]

File: client/src/App.js
Comment (Adibab) [reply client/src/App.js:1]: No, I dont want to change it. Now I have sorted out the router issue. Thanks

File: client/components/users/VerifyCertificateNum.js (hunk 6 above)
Comment (Adibab) [reply client/components/users/VerifyCertificateNum.js]: Hi, I have amended it. thanks.

File: package.json
Comment (Rafsanjani) [inline package.json:1]: Run `npm install` locally and commit your `package-lock.json` file

**Review by Rafsanjani — APPROVED** [review]

**Comment by Rafsanjani** [comment]
Can you merge main back into this branch

**Pull request merged**